docs/advanced.md

```diff
@@ -57,6 +57,28 @@ dispatch = httpx.WSGIDispatch(app=app, remote_addr="1.2.3.4")
 client = httpx.Client(dispatch=dispatch)
 ```
 
+## Build Request
+
+Using a Client instance to make build requests
+```python
+>>> client = httpx.Client()
+>>> r = client.build_request("GET", "https://example.com")
+>>> r
+<AsyncRequest('GET', 'https://example.com')>
+>>> client.send(r)
+<Response [200 OK]>
+```
+And async client
```

Comment:
Let's not have a duplicate demonstration for the AsyncClient, client and async client have the same interface, only show where they differ.